.github/workflows/lint-and-test.yml

```diff
@@ -130,6 +130,8 @@ jobs:
         run: cargo run --example plastics
       - name: Run sushi example
         run: cargo run --example sushi
+      - name: Run stocks example
+        run: cargo run --example stocks
```

Comment:
This change is no longer needed and can safely be reverted

Reply:
reverted